apps/dashboard/src/components/hackathon/HacakthonEarnFooter.tsx

```diff
@@ -60,7 +60,7 @@ export const HackathonEarnFooter = ({
           h="68px"
           w={{ base: "90%", md: 80 }}
           fontSize="20px"
-          leftIcon={<Icon as={ImMagicWand} />}
+          leftIcon={<Wand />}
```

Comment:
size class missing, also prefer the name with "Icon` postfix - eg: `WandIcon` here